core-graphics/src/access.rs

```diff
@@ -14,7 +14,7 @@ impl ScreenCaptureAccess {
     /// Return `true` if has access
     #[inline]
     pub fn preflight(&self) -> bool {
-        unsafe { CGPreflightScreenCaptureAccess() == 1 }
+        unsafe { (CGPreflightScreenCaptureAccess() & 1) == 1 }
```

Comment:
Won't this return false if CGPreflightScreenCaptureAccess() returns 2?
Should request() also be changed.

Can you also add some more documentation about what semantics you expect CGPreflightScreenCaptureAccess() to have. i.e. what are the non 1 values you see it returning.

Reply:
> i.e. what are the non 1 values you see it returning.

I don't remember the exact value, but it was a 16bit number where the last 8 bits were `00000001`

> Won't this return false if CGPreflightScreenCaptureAccess() returns 2?

Yes. It would. But I think 2 won't be returned.

>Should request() also be changed.

Yup. I'll do the change if this change looks good. `preflight` was on a higher priority so I did this change first.